### ISSUE TITLE
Action Recognition in Video Sequences using Deep Bi-Directional LSTM With CNN Features

### ISSUE THREAD

**Issue by Toraudonn** [Toraudonn/DL-Paper-Challenge#2]
## [Action Recognition in Video Sequences using Deep Bi-Directional LSTM With CNN Features](https://ieeexplore.ieee.org/document/8121994/)

- Authors: Ullah A, Ahmad J, Muhammad K, Sajjad M, Baik S
- Conference: IEEE Published
- Code:
- Year: 2017

## Summary

**Comment by Toraudonn** [comment]
Conventional methods:

- Hand-crafted:
    - extract low-level features and input the features to classifiers such as SVM, decision tree, and KNN
    - What kind of features?
        - stacked body contours (geometrical properties called [space time volume (STV)](http://www.csd.uwo.ca/~ygorelic/spaceTimeActionsTPAMI2007.pdf))
        - [motion history image and histogram of oriented gradients feature (HOG)](https://pdfs.semanticscholar.org/cc87/45e10489f52ffd4da32ab3df63cbf16e5225.pdf)

![screen shot 2018-10-29 at 15 11 19](https://user-images.githubusercontent.com/7902640/47632320-ecd3ad80-db8c-11e8-9caa-6adf84e72877.png)

- Deep learning based methods:
    - 3D convolutional network
    - multi-resolution CNN frameworks
    - Pre-trained CNN feature extractor

**Comment by Toraudonn** [comment]
![image](https://user-images.githubusercontent.com/7902640/47633672-20fd9d00-db92-11e8-875f-e494f2d9fc96.png)

- Bidirectional LSTM
    - [reference](https://stackoverflow.com/questions/43035827/whats-the-difference-between-a-bidirectional-lstm-and-an-lstm)
    - RNN -> used for analyzing hidden sequential patterns in both temporal sequential and spatial sequential data
    - RNN has the [vanishing gradient problem](https://www.quora.com/What-is-the-vanishing-gradient-problem) which LSTM solves
    - Complex sequence patterns (such as video data) are not identified by the single LSTM cell -> use a multi-layer LSTM
        - all sequential output of the first LSTM is fed into the next LSTM layer
    - Bidirectional LSTM means there are two LSTM's which 1 LSTM computes forward and another one goes backward

![screen shot 2018-10-29 at 16 09 32](https://user-images.githubusercontent.com/7902640/47634491-0ed12e00-db95-11e8-93a5-a4dfeb7a90af.png)
